install.php

```diff
@@ -24,11 +24,21 @@
             </ul>
         </li>
     </ol>
-
+    
+    <h2 class="hdr" id="quickstart">Upgrading to a new major release of macOS</h2>
+    
+    <ol>
+        <li>Install the latest version of <a href="<?php print $guide_url . '#installing.xcode'; ?>">Xcode and the Xcode Command Line Tools</a></li>
+        </li><a href="<?php print $trac_url . '/wiki/Migration#config'; ?>">Update your macports.conf</a> (if not default)</li>
```

Comment:
Surely we should be consistent about whether or not there is a `/` before `wiki`. I am guessing that there should not be a slash.

You have a closing `</li>` tag at the beginning of this line that should be an opening tag instead.